Library/Formula/fish.rb

```diff
@@ -36,9 +36,16 @@ def caveats; <<-EOS.undent
       chsh -s #{HOMEBREW_PREFIX}/bin/fish
     to make fish your default shell.
 
-    If you are upgrading from an older version of fish, you should now run:
+    And optionally:
+      fish_config
```

Comment:
> Is this specific to Homebrew?

No; in fact, fish on first run prints a greeting,
```
Welcome to fish, the friendly interactive shell.
Type help for instructions on how to use fish.
```
Which really is more that sufficient to cover this.